code/game/mecha/mecha_construction_paths.dm

```diff
@@ -21,7 +21,7 @@
 	if(!ispath(step[key]))
 		if(istype(used_atom, /obj/item))
 			var/obj/item/I = used_atom
-			return I.use_tool(user, holder, WORKTIME_NORMAL, step[key], FAILCHANCE_NORMAL, required_stat = STAT_MEC)
+			return I.use_tool(user, holder, WORKTIME_EXTREMELY_LONG, step[key], FAILCHANCE_NORMAL, required_stat = STAT_MEC)
```

Comment:
This is dumb ill prob revert this and add something better. `Man screwing in a single board into a case takes 4 mins...`